ami/models/hifigan_generator.py

```diff
@@ -0,0 +1,175 @@
+# Ref:
+# https://github.com/jik876/hifi-gan/blob/master/models.py
+
+import torch
+from torch.nn import Conv1d, ConvTranspose1d
+
+
+def init_weights(m: torch.nn.Module, mean: float = 0.0, std: float = 0.01) -> None:
+    classname = m.__class__.__name__
+    if classname.find("Conv") != -1:
+        m.weight.data.normal_(mean, std)
+
+
+def get_padding(kernel_size: int, dilation: int = 1) -> int:
+    return int((kernel_size * dilation - dilation) / 2)
+
+
+class ResBlock1(torch.nn.Module):
+    def __init__(self, channels: int, kernel_size: int = 3, dilation: list[int] = [1, 3, 5]) -> None:
+        super().__init__()
+        self.convs1 = torch.nn.ModuleList(
+            [
+                torch.nn.utils.weight_norm(
```

Comment:
deprecation warning
https://pytorch.org/docs/2.1/generated/torch.nn.utils.weight_norm.html#torch.nn.utils.weight_norm
```suggestion
                torch.nn.utils.parametrizations.weight_norm(
```

Reply:
https://github.com/MLShukai/ami/commit/d95b59e0287e56352e913ae1527480886d3afb44 で修正